test/JDBC/expected/temp_table_rollback-vu-verify.out

```diff
@@ -153,37 +153,21 @@ BEGIN TRAN
 ALTER TABLE #temp_table_rollback_t1 DROP COLUMN b
 ROLLBACK
 GO
-~~ERROR (Code: 3726)~~
-
-~~ERROR (Message: cannot drop column b of table "#temp_table_rollback_t1" because other objects depend on it)~~
-
```

Comment:
thanks for handling this.
Do we have tests for scenarios where it's supposed to fail with this error?

Reply:
I just added an extra test case, that while not reproducing this exact error, produces a similar error due to a dependency.